### PR TITLE
#173 minor updates on doku

### LINKED ISSUE #173: Tutorial Improvements
I was just going through the documentation and I think we could slightly improve the tutorial. 

Following things I could think of:
- Describing the use case we are showing
- Describing the artifacts we are using/mocking in this tutorial (RIG, the frontend, the backend)
- Clearly mention for each step which artifact we're working on right now
- Remove step 6 from the workflow as it actually doesn't fit there. It should be mentioned that this is how you would implement above's steps in a JS frontend?

What do you guys think? @kevinbader @mmacai

### PR DESCRIPTION
**Pull request by jakobwgnr**
closes #173

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ title: Tutorial
 sidebar_label: Tutorial
 ---
 
-In this tutorial we use [HTTPie](https://httpie.org/) for HTTP requests, but of course you can also use curl or any other HTTP client. Please note that HTTPie sets the content type to `application/json` automatically, whereas for curl you need to use `-H "Content-Type: application/json"` for all but `GET` requests.
+This tutorial should show a basic usecase for RIG. A frontend (e.g. the mobile app for a chatroom service) will connect to RIG and subscribe to a certain eventtype (e.g. Messages from a chatroom channel). The backend (e.g. chatroom server) will publish the message to RIG and the RIG will do the rest.
```
Comment (kevinbader) [inline docs/tutorial.md]:
```suggestion
This tutorial shows a basic use case for RIG. A frontend (e.g. the mobile app for a chatroom service) connects to RIG and subscribes to a certain event type (e.g. messages from a chat room). The backend (e.g. chat-room server) publishes the message to RIG, and RIG forwards it to the frontend.
```

File: docs/tutorial.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ title: Tutorial
...
+
+We'll fake frontend/backend using [HTTPie](https://httpie.org/) for HTTP requests, but of course you can also use curl or any other HTTP client. Please note that HTTPie sets the content type to `application/json` automatically, whereas for curl you need to use `-H "Content-Type: application/json"` for all but `GET` requests.
```
Comment (kevinbader) [inline docs/tutorial.md]:
```suggestion
We simulate frontend and backend HTTP requests using [HTTPie](https://httpie.org/), but of course you can also use curl or any other HTTP client. Please note that HTTPie sets the content type to `application/json` automatically, whereas for curl you need to use `-H "Content-Type: application/json"` for all but `GET` requests.
```

File: docs/tutorial.md [hunk 3]
```diff
@@ -41,22 +43,22 @@ After the connection has been established, RIG sends out a [CloudEvent](https://
...
 
-## 3. Subscribe to a topic
+## 3. Subscribe to a topic [Frontend]
 
 With the connection established, you can create _subscriptions_ - that is, you can tell RIG which events your app is interested in. RIG needs to know which connection you are referring to, so you need to use the connection token you have noted down in the last step:
 
 ```bash
 $ CONN_TOKEN="g2dkAA1ub25vZGVAbm9ob3N0AAACrAAAAAAA"
-$ SUBSCRIPTIONS='{"subscriptions":[{"eventType":"greeting"}]}'
+$ SUBSCRIPTIONS='{"subscriptions":[{"eventType":"chatroom_channel_messages"}]}'
```
Comment (kevinbader) [inline docs/tutorial.md]: `chatroom_channel_messages` => `chatroom_message`? The event type is often singular and the `channel` in there doesn't seem to add any meaning.

File: docs/tutorial.md [hunk 4]
```diff
@@ -87,13 +89,15 @@ RIG responds with `202 Accepted`, followed by the CloudEvent as sent to subscrib
...
-## 5. The event has been delivered to our subscriber
+## 5. The event has been delivered to our subscriber [Frontend]
+
+Going back to the first terminal window you should now see your greeting event
 
-Going back to the first terminal window you should now see your greeting event :tada:
+# Connect your app to RIG
 
-## 6. Connect your app to RIG
+In a real-world frontend app the above example to connect your app to RIG would look something like this below.
```
Comment (kevinbader) [inline docs/tutorial.md]: thought you wanted to get rid of the code below as well.. no longer the case?

Comment (jakobwgnr) [reply docs/tutorial.md]: I wanted to keep, but for me it was just a bit misleading as it was numbered :)

**Comment by jakobwgnr** [comment]
@kevinbader 
Updates done

**Pull request merged**